pkg/apis/healthchecking/v1alpha1/machinehealthcheck_types.go

```diff
@@ -4,6 +4,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// ConfigMapNodeUnhealthConditions contains the name of the unhealth conditions config map
+const ConfigMapNodeUnhealthConditions = "node-unhealth-conditions"
```

Comment:
can we use unhealthy instead consistently?

Reply:
done